docs/conceptual/gpu-arch/mi200-performance-counters.md

```diff
@@ -1,455 +1,591 @@
+| `TCC_EA_ATOMIC[n]` | Req | Number of 32-byte or 64-byte atomic requests going over the `TC_EA_wrreq` interface (value range for n: [0-31]) |
+| `TCC_EA_ATOMIC_LEVEL[n]` | Req | Accumulated number of EA atomic requests in flight, primarily intended to measure average EA atomic latency; average atomic latency = `TCC_PERF_SEL_EA_WRREQ_ATOMIC_LEVEL`/`TCC_PERF_SEL_EA_WRREQ_ATOMIC` (value range for n: [0-31]) |
+| `TCC_EA_RDREQ[n]` | Req | Number of 32-byte or 64-byte read requests to EA (value range for n: [0-31]) |
+| `TCC_EA_RDREQ_32B[n]` | Req | Number of 32-byte read requests to EA (value range for n: [0-31]) |
+| `TCC_EA_RD_UNCACHED_32B[n]` | Req | Number of 32-byte EA reads due to uncached traffic; a 64-byte request is counted as 2 (value range for n: [0-31]) |
+| `TCC_EA_RDREQ_IO_CREDIT_STALL[n]` | Cycles | Number of cycles there is a stall due to the read request interface running out of IO credits; stalls occur irrespective of the need for a read to be performed (value range for n: [0-31]) |
+| `TCC_EA_RDREQ_GMI_CREDIT_STALL[n]` | Cycles | Number of cycles there is a stall due to the read request interface running out of GMI credits; stalls occur irrespective of the need for a read to be performed (value range for n: [0-31]) |
+| `TCC_EA_RDREQ_DRAM_CREDIT_STALL[n]` | Cycles | Number of cycles there is a stall due to the read request interface running out of DRAM credits; stalls occur irrespective of the need for a read to be performed (value range for n: [0-31]) |
+| `TCC_EA_RDREQ_LEVEL[n]` | Req | The accumulated number of EA read requests in flight, primarily intended to measure average EA read latency; average read latency = `TCC_PERF_SEL_EA_RDREQ_LEVEL`/`TCC_PERF_SEL_EA_RDREQ` (value range for n: [0-31]) |
+| `TCC_EA_RDREQ_DRAM[n]` | Req | Number of 32-byte or 64-byte EA read requests to HBM (value range for n: [0-31]) |
+| `TCC_EA_WRREQ_DRAM[n]` | Req | Number of 32-byte or 64-byte EA write requests to HBM (value range for n: [0-31]) |
+| `TCC_TAG_STALL[n]` | Cycles | Number of cycles the normal request pipeline in the tag is stalled for any reason--note that stalls are typically measured exactly at one point in the pipeline; however, with this counter, probes can stall the pipeline at a variety of places and there is no single point that can reasonably measure the total stalls accurately (value range for n: [0-31]) |
+| `TCC_NORMAL_WRITEBACK[n]` | Req | Number of writebacks due to requests that are not writeback requests (value range for n: [0-31]) |
+| `TCC_ALL_TC_OP_WB_WRITEBACK[n]` | Req | Number of writebacks due to all `TC_OP` writeback requests (value range for n: [0-31]) |
+| `TCC_NORMAL_EVICT[n]` | Req | Number of evictions due to requests that are not invalidate or probe requests (value range for n: [0-31]) |
+| `TCC_ALL_TC_OP_INV_EVICT[n]` | Req | Number of evictions due to all TC_OP invalidate requests (value range for n: [0-31]) |
 
 ## MI200 derived metrics list
```

Comment:
@SwRaw this is an empty heading; We should add content here or remove the following heading.